### PR TITLE
Routing skeleton

### PR DESCRIPTION
**Pull request by KainyRyu**
Created the skeleton of routing and the whole structure

### CODE REVIEW

File: src/frontend/components/Header/Header.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-      <div className="Header-top">
+      <div className="navbar">
         <div className="logo">
           <img className="logo-img" src={logo} alt="logo" />
         </div>
-        <div className="signing">
-          <a className="sign-up">Sign Up </a>
+        <div className="navbar-div">
+          <span className="sign-up" >
+            Sign Up{" "}
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/Header/Header.js]: why do we need `{" "}` in the text ?

File: src/frontend/components/Result/Results.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Results.css';
+import SearchBar from '../SearchBar/SearchBar';
+import people from '../../image/people.png';
+
+
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/Result/Results.js]: can remove blank lines

File: src/frontend/components/Result/Results.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                    <img className="user-image" src={people} alt="people" />
+                    <div>dummy Name</div>
+                    </td>
+                    <td>dummy Language</td>
+                    <td>dummy Location</td>
+                </tr>
+            </table>
+        </div>
+    )
+}
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/Result/Results.js]: nice

File: src/frontend/components/RoutingPage/RoutingPage.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+export default function RoutingPage() {
+  return (
+    <div className="App">
+      <Header />
+      <main>
+        <Search />
+        <button clascsName='searchbtn'>Search</button>
+      </main>
+    </div>
+
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/RoutingPage/RoutingPage.js]: blank line

File: src/frontend/components/Header/Header.js (hunk 1 above)
Comment (KainyRyu) [reply src/frontend/components/Header/Header.js]: That's a space in between sign up | long in 
maybe when we do the UI we can change it!

File: src/frontend/components/Result/Results.js (hunk 2 above)
Comment (KainyRyu) [reply src/frontend/components/Result/Results.js]: Yes

**Pull request merged**